blog/langlydz-part-ilewen/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Langłydż - part ilewen: żargon, zapożyczenia i polglisz'
+authors: pensjonatus
+date: '2024-12-21'
+tags:
+  - 'warsztat'
+  - 'z-przymrużeniem-oka'
+  - 'humor'
+  - 'język'
+coverImage: 'polglisz.png'
+---
+
+Ludzie czasem hejtują na korpomowę. Ja tego nie robię i to serio, bez ironii.
+Korpomowa to nie jest jakaś orwellowska nowomowa. Korpomowa to coś bardzo
+przydatnego: **żargon**.
+
+<!--truncate-->
+
+## Po co jest żargon
+
+![Dwaj przyjaciele na wakacjach, jeden mówi "co mamy dzisiaj na agendzie?" a drugi odpowiada śmiejąc się: "hahaha, nie wiem"](./images/agenda.png)
+
+Żargon, w tym żargon oparty o zapożyczenia, jest jak najbardziej w porządku.
+Ułatwia i przyspiesza komunikację, a dodatkowo służy do identyfikacji zawodowej
+(mówimy i czujemy się jak członkowie grupy). Żargon jest ładny, tak jak żółty
+kask jest ładny: bo pasuje do okazji. Na budowie używasz żółtego kasku, a w
+pracy używasz żargonu.
+
+Przykład: Robimy PRy do repo żeby rozwiązać issue. Jak nafarmimy quick winów, to
+będzie o czym opowiadać na meetupie. Dobrze się składa, bo otworzyli call for
+papers.
+
+Jak jest _call for papers_ po polsku? Nie wiem, bo nie brałem udziału w
+konferencji akademickiej od czasów studiów. Szybciej jest powiedzieć call for
+papers, niż tworzyć sztuczne zamienniki, albo robić yyy i aaa próbując sobie
+przypomnieć jakieś archaiczne polskie słowo.
+
+## Przepraszam, czy mogę pożyczyć słówko?
+
+Zapożyczenia, nie tylko żargonowe, ale również te w mowie powszechnej, ogólnie
+wzbogacają język. Bez nich byśmy nie mieli fryzjera, dachu, czy meczu.
+
+Nawet zapożyczone znaczenia są fajne; kontrybucja nie musi się kojarzyć tylko z
+wojną, ale może oznaczać pozytywny wkład. Pasjonat nie musi wpadać w złość, może
+po prostu bardzo coś lubić. A kolaboracja nie musi oznaczać przejścia na stronę
+wroga.
+
+Jestem po studiach językoznawczych i mam silne przekonanie, że mój punkt
+widzenia nie wymaga obrony. To już było, to już przerabialiśmy,, wiele wiele
+razy. Zapożyczenia sprawiają, że język żyje.
+
+Polglisz też na pewno wzbogaca język, co nie zmienia faktu, **że mam kilka
+pytań**. Dlatego też napisałem ten mini artykulik: bo mam kilka pytań o
+polglisz.
+
+![dwie koleżanki na kanapie, ta po lewej płacze. ta po prawej pyta: "Marta, jak mogę zaadresować twój problem?" a ta po lewej odpowiada: "zaadresować to sobie możesz kartkę do cioci, hehe... chlip... chlip..."](./images/zaadresowac.png)
+
+## Pytania o polglisz
+
+Jest taka platforma shoper.pl. Czy poprawna wymowa to _szołper_?
```

Comment:
mnie zawsze zastanawia sklep Half Price.

W reklamach sklepu wymawiaja HALF price, a ja chcialbym to wymowic zgodnie z zasadami wymowy Posh English "HAF PRICE". Dyskutuje z obsluga sklepu na ten temat i nie mozemy dosc do konsensusu ;)

Moze pojsc po calosci i wymawiac calosc fonetycznie, bo jednak drugie slowo wymawiaja PRAJS ;)

I'm soo confused...